lib/models/Bot.js

```diff
@@ -368,7 +371,33 @@ class Bot extends BaseModel {
         return output;
     }
 
-    async generateVisor(parentId, role, conflicts, listPredefinedItems = false) {
+    async generateLootInPockets(pocketContainer, parentId = this.generateInventoryItems.parentId)
+    {
+        const output = [];
+        //The IDs of various items in ItemEntries are subject to change! These are just test Items.
+        const ItemEntries = [
+            "544fb45d4bdc2dee738b4568", //Salewa
+            "5d02778e86f774203e7dedbe", //Core Medical Surgical Kit
+            "590c5f0d86f77413997acfab",  // MRE Ration Pack
+            "5909d24f86f77466f56e6855" // Medbag
+        ]
+        const newItemTemplate = await Item.get(await getRandomFromArray(ItemEntries));
+        const itemSize = await newItemTemplate.getSize();
+        const newItem = await newItemTemplate.createAsNewItem();
+        const upd = await Item.createFreshBaseItemUpd(newItemTemplate);
+        const freeSlot = await Item.getFreeSlot(pocketContainer, output, itemSize.width, itemSize.height);
+        if(freeSlot)
```

Comment:
This will give you an error as i believe, or not idk, you could add check if undefined or smth and return empty array